product/views.py

```diff
@@ -0,0 +1,61 @@
+import json
+import math
+
+from django.shortcuts      import render
+from django.views          import View
+from django.http           import JsonResponse, HttpResponse
+from django.db.models      import Count
+
+from .models               import Product, Category, Image, Option, Question, Review, Comment
+
+# 상품 전체 보기
+class ProductAllView(View):
+    def get(self, request):
+        try:
+            page         = int(request.GET.get('page', 1))
+            page_size    = 9
+            max_page     = math.ceil(Product.objects.all().count()/9)
```

Comment:
```suggestion
            max_page     = math.ceil(Product.objects.all().count()/page_size)
```
이렇게 하면 하드코딩이 아니라 좋을것 같아요.

Reply:
상수로 바꾸면 상수로 입력해주세요. (PAGE_SIZE)

Reply:
네네!!